com.epicgames.ut99.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: com.epicgames.ut99
+tags:
+  - proprietary
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: ut99
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
```

Comment:
Does not work natively on Wayland. No game window is displayed when a wayland socket is enabled.